Add Ecosystem Sustainability Program (ESP) Page (#1309)
openjs-foundation/cross-project-council: project-resources/ESP/ECOSYSTEM_SUSTAINABILITY_PROGRAM.md

# OpenJS Ecosystem Sustainability Program 

## Summary 

> The Ecosystem Sustainability Program is an opt-in OpenJS partner program to help secure outdated software and create new revenue streams for our projects and commercial participants.  

Reviewer: ```suggestion
The Ecosystem Sustainability Program is an opt-in OpenJS partner program to help secure outdated software and create new revenue streams for our projects.  
```

Same.

## Benefits

Eligible projects that opt into the program receive revenue that can be used to:

- Grow their community
- Pay contractors
- Support travel
- Cover other project-related expenses

## Current Partners

- [HeroDevs](https://www.herodevs.com/)

## Eligibility

For a project to participate in the program:

> - A partner must be providing support to EOL versions of the project

Reviewer: ```suggestion
- A partner must be providing support end-of-life (EOL) versions of the project
```

alt: “no longer supported”

> - The project must be willing to place links to the partner on various pages/repositories where EOL versions are mentioned (requirements will vary per partner)

Reviewer: How does the “requirements will vary per partner” work in practice? It feels like this should be negotiated by the foundation for all of the projects and partners and not the fruit of 1:1 negotiations between projects and partners.

Reviewer: Also worth linking to the section below where more info is provided.

- The project must be willing to manage funds via [Open Collective](https://opencollective.com/).

## How To Participate

Projects that are interested in participating in the program should contact operations@openjsf.org.

## How To Enroll In Open Collective

Because funds are distributed to the project and in most cases not an individual, we suggest creating an *[Open Collective](https://docs.opencollective.com/help/collectives/create-collective)* account and project collective. 

> In most cases, projects do not have bank accounts and must [sign up](http://opencollective.com/opensource/apply) with Open Source Collective to be a fiscal host. This means Open Collective will hold the funds on your behalf, handle taxes, and enable you to pay vendors and contractors. 

Reviewer: ```suggestion
In most cases, projects do not have bank accounts and must [sign up](https://opencollective.com/opensource/apply) with Open Source Collective to be a fiscal host. This means Open Collective will hold the funds on your behalf, handle taxes, and enable you to pay vendors and contractors. 
```

### Important Information About Payments

- Payments are made to projects every 6 months

> - Project payments are excluded from the LF G&A fee

Reviewer: I have no idea what the LF G&A fee means

Reviewer 2: https://cdn.platform.linuxfoundation.org/agreements/pqca.pdf

> A General & Administrative (G&A) fee will be applied by The Linux Foundation to funds raised to cover membership records, finance, accounting, and human resources operations. The G&A fee will be 9% of the Directed Fund’s first $1,000,000 of gross receipts each year and 6% of the Directed Fund’s gross receipts each year over $1,000,000.

Reviewer: Is this something the OpenJS is subject to? If so, let’s spell acronym out and link it.

Reviewer 2: Yes, as far as i'm aware. linking sounds good.

Author: Will add link when I find it :)

Reviewer 2: just linking to the PDF i linked above and referencing the section number should be fine
